content/2_Bronze/Conclusion.mdx

```diff
@@ -0,0 +1,35 @@
+---
+id: bronze-conclusion
+title: Additional Practice for USACO Bronze
+author: Nathan Wang
+description:
+  'Congratulations on making it this far! Here are some final tips for Bronze,
```

Comment:
I think would be great if each division has a different "congratulations paragraph" something fun and/or representative of each division like if you pass bronze "farmer Nhoj will allow you to milk Bessie" and in silver "you are now allowed to drink the milk". It could be fun :P